low-level/framework/utils/server_drives.py

```diff
@@ -0,0 +1,214 @@
+# Copyright (c) 2001-2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
Can we put this under 
low-level/framework/platform/server/drives

Lets start building the hierarchy classes

Reply:
Done